public/js/index.js

```diff
@@ -117,6 +117,25 @@ function updateIssuesMeta(url, action, issueIds, elementId, afterSuccess) {
     })
 }
 
+function initReactionSelector(parent) {
+    var container = (parent ? parent + ' ' : '');
+    $(container + '.select-reaction > .menu > .item').on('click', function(e){
+        e.preventDefault();
+        var url = $(this).closest('.select-reaction').data('action-url')
+            + '/' + ($(this).hasClass('is-active') ? 'unreact' : 'react');
```

Comment:
This `is-active` class isn't used anywhere, so the action is always `react`, and it's impossible to unreact.